ChiModules/LinearBoltzmannSolver/lbs_linear_boltzmann_solver.h

```diff
@@ -30,11 +30,12 @@ enum class BoundaryType
 };
   enum SourceFlags : int
   {
-    NO_FLAGS_SET           = 0,
-    APPLY_MATERIAL_SOURCE  = (1 << 0),
+    NO_FLAGS_SET               = 0,
+    APPLY_MATERIAL_SOURCE      = (1 << 0),
     APPLY_WGS_SCATTER_SOURCE   = (1 << 1),
     APPLY_AGS_SCATTER_SOURCE   = (1 << 2),
-    APPLY_FISSION_SOURCE       = (1 << 3)
+    APPLY_WGS_FISSION_SOURCE   = (1 << 3),
+    APPLY_AGS_FISSION_SOURCE   = (1 << 4)
   };
```

Comment:
As mentioned by others, this needs to be documented.

Reply:
This is true, but these were accepted without documentation prior to this PR in one authored not by myself. The adoption of the bit-wise flags is not really within the scope of this PR and I think would be better addressed by raising an issue.

Reply:
This was raised as an issue in #165